manage-your-labels.yml

```diff
@@ -0,0 +1,117 @@
+LABEL_GROUPS:
+    DEFAULT:
+        bug:
+            description: Something isn't working
+            color: d73a4a
+        debt:
+            description: Issues to deal with later
+            color: 0c0c75
+        dependencies:
+            description: Pull requests that update a dependency file
+            color: 0366d6
+        documentation:
+            description: Improvements or additions to documentation
+            color: efef15
+        duplicate:
+            description: This issue or pull request already exists
+            color: cfd3d7
+        enarxbot:
+            description: Issues or feature requests for the Enarx project automation suite
+            color: D3A960
+        enhancement:
+            description: New feature or request
+            color: a2eeef
+        expertise needed:
+            description: This needs special attention from an area specialist.
+            color: f9928e
+        good first issue:
+            description: Good for newcomers
+            color: a8f475
+        help wanted:
+            description: Extra attention is needed
+            color: '008672'
```

Comment:
Are the quotes around this value significant?

Reply:
@connorkuehl I believe it's because all the characters are digits, so if it's not quoted it'll be interpreted as an integer instead of a string.